core/src/contract_context.rs

```diff
@@ -49,7 +49,7 @@ pub trait ContractContext {
     ///
     /// * `dictionary_name` - The name of the dictionary.
     /// * `key` - The key to retrieve the value for.
-    fn get_dictionary_value(&self, dictionary_name: &str, key: &str) -> Option<Bytes>;
```

Comment:
Make sure this works by running a livenet example of cep-18 against original contract implementation.